scaleway/resource_rdb_instance.go

```diff
@@ -442,7 +449,7 @@ func resourceScalewayRdbInstanceUpdate(ctx context.Context, d *schema.ResourceDa
 	if d.HasChange("backup_schedule_retention") {
 		req.BackupScheduleRetention = scw.Uint32Ptr(uint32(d.Get("backup_schedule_retention").(int)))
 	}
-	if d.HasChange("tags") {
+	if d.HasChangeExcept("tags") {
```

Comment:
Why this change is required?

Reply:
Unfortunately not. I'll add a new test and keep HasChange